dataframe.ipynb

```diff
@@ -258,7 +267,15 @@
    "metadata": {},
    "outputs": [],
    "source": [
-    "df[['x', 'y']].resample('24h').mean().compute().plot()"
+    "%matplotlib inline\n",
```

Comment:
Are these still necessary? I haven't needed to run `%matplotlib inline` in a long time.

Reply:
I don't know if they're needed anymore, I didn't intentionally add or remove any lines like this.